### PR TITLE
Niad 2122 ambulance input

### PR DESCRIPTION
**Pull request by hodge70**

### CODE REVIEW

File: test-suite/client/src/data/schema.ts [hunk 1]
```diff
@@ -692,7 +692,245 @@ const schema: Schema = {
...
+                        validators: [notNull()]
+                      }
+                    ]
+                  }
+                },
+                {
+                  testName: "Ambulance input",
+                  testDescription:
+                    "This test is for an ambulance medic to supply input on transit",
+                  template: require("./ambulance-itk-request.xml"),
```
Comment (SRAlexander) [inline test-suite/client/src/data/schema.ts:852]: This needs to be added to the commit

**Review by SRAlexander — CHANGES_REQUESTED** [review]

File: test-suite/client/src/data/schema.ts (hunk 1 above)
Comment (hodge70) [reply test-suite/client/src/data/schema.ts:852]: Added idk file as discussed

File: test-suite/client/src/data/schema.ts [hunk 2]
```diff
@@ -692,7 +692,245 @@ const schema: Schema = {
...
+                      },
+                      {
+                        label: "Recipient DOS Code",
+                        id: "dos-code",
+                        defaultValue: "159744",
+                        validators: [isAlphanumeric()]
+                      },
+                      {
+                        label: "Recipient Name",
+                        id: "recipient-name",
```
Comment (SRAlexander) [inline test-suite/client/src/data/schema.ts:737]: Does not appear in ITK message.

Could you update the ITK message with the missing field, other ITK messages should have the field within them... syntax should be along the lines of @@id@@.

File: test-suite/client/src/data/schema.ts [hunk 3]
```diff
@@ -692,7 +692,245 @@ const schema: Schema = {
...
+                        defaultValue: "Bramley Road Surgery",
+                        validators: [notNull(), isAlpha()]
+                      },
+                      {
+                        label: "Recipient Address Line 1",
+                        id: "recipient-street-address-line-1",
```
Comment (SRAlexander) [inline test-suite/client/src/data/schema.ts:743]: Does not appear in ITK message.

Could you update the ITK message with the missing field, other ITK messages should have the field within them... syntax should be along the lines of @@id@@.

File: test-suite/client/src/data/schema.ts [hunk 4]
```diff
@@ -692,7 +692,245 @@ const schema: Schema = {
...
+                        defaultValue: "Bramley Road Surgery",
+                        validators: [notNull(), isAlphanumeric()]
+                      },
+                      {
+                        label: "Recipient Address Line 2",
+                        id: "recipient-street-address-line-2",
```
Comment (SRAlexander) [inline test-suite/client/src/data/schema.ts:749]: Does not appear in ITK message.

Could you update the ITK message with the missing field, other ITK messages should have the field within them... syntax should be along the lines of @@id@@.

File: test-suite/client/src/data/schema.ts [hunk 5]
```diff
@@ -692,7 +692,245 @@ const schema: Schema = {
...
+                      },
+                      {
+                        label: "Recipient City",
+                        id: "recipient-street-address-city",
+                        defaultValue: "London",
+                        validators: [isAlpha()]
+                      },
+                      {
+                        label: "Recipient Postcode",
+                        id: "recipient-street-address-postcode",
```
Comment (SRAlexander) [inline test-suite/client/src/data/schema.ts:767]: Does not appear in ITK message.

Could you update the ITK message with the missing field, other ITK messages should have the field within them... syntax should be along the lines of @@id@@.

File: test-suite/client/src/data/schema.ts (hunk 4 above)
Comment (SRAlexander) [reply test-suite/client/src/data/schema.ts:749]: * Placed on wrong line, this is for the case below

File: test-suite/client/src/data/schema.ts [hunk 6]
```diff
@@ -692,7 +692,245 @@ const schema: Schema = {
...
+                      },
+                      {
+                        label: "Recipient Town",
+                        id: "recipient-street-address-town",
+                        defaultValue: "Ealing",
+                        validators: [isAlpha()]
+                      },
+                      {
+                        label: "Recipient City",
+                        id: "recipient-street-address-city",
```
[the remark this answers is not in the document]
Comment (SRAlexander) [reply test-suite/client/src/data/schema.ts:761]: * Placed on wrong line, this is for the case below

**Review by SRAlexander — CHANGES_REQUESTED** [review]

**Pull request merged**